### PR TITLE
Enhance icoxxlist plotter

### PR DESCRIPTION
**Pull request by naik-aakash**
Add option to color icoxxlist plots based on atom pair interactions

NaCl

### CODE REVIEW

File: src/lobsterpy/plotting/__init__.py
Comment (QuantumChemist) [inline src/lobsterpy/plotting/__init__.py:1096]: what do you think about a customizable color palette?

**Comment by naik-aakash** [comment]
Hi @JaGeo , can be merged if this output form is fine.

File: src/lobsterpy/plotting/__init__.py
Comment (naik-aakash) [reply src/lobsterpy/plotting/__init__.py:1096]: Is it there, in get_plot one can pass list of custom hex colors and it will use that then 😄

**Review by JaGeo — APPROVED** [review]

**Comment by JaGeo** [comment]
Ah, @naik-aakash : i forgot. it is not yet available via the command line, right?

**Comment by naik-aakash** [comment]
> Ah, @naik-aakash : i forgot. it is not yet available via the command line, right?

Yes, I have this on my todo and will add option to this in cli once we merge the bwdf featurizer, as I assume we would like to have plots as well from there. 

One PR to update the CLI

**Comment by JaGeo** [comment]
Put this as an issei. Fine for now.

File: src/lobsterpy/plotting/__init__.py
Comment (QuantumChemist) [reply src/lobsterpy/plotting/__init__.py:1096]: cool, didn't see it x)

**Pull request merged**
